content-review/impl/turnitin-oc/src/main/java/org/sakaiproject/contentreview/turnitin/oc/ContentReviewServiceTurnitinOC.java

```diff
@@ -1003,26 +960,149 @@ private Date getDueDateRetryTime(Date dueDate) {
 		return cal.getTime();
 	}
 
+	private boolean checkForDraft(ContentReviewItem item, Assignment assignment) {
+		// Checks if current item is a draft or submitted
+		try {
+			AssignmentSubmission currentSubmission = assignmentService.getSubmission(assignment.getId(), item.getUserId());
+			// Drafts return true, final submissions return false, if null return false
+			return Optional.ofNullable(!currentSubmission.getSubmitted()).orElse(false);
```

Comment:
this doesn't check the item status, only whether the student submitted the assignment as a whole